datumaro/components/operations.py

```diff
@@ -78,6 +78,13 @@ def merge_annotations_equal(a, b):
     return [ann_a for (ann_a, _) in matches] + a_unmatched + b_unmatched
 
 
+def merge_infos(sources):
+    infos = {}
+    for source in sources:
+        infos.update(source)
```

Comment:
I think that if the same key exists in `infos`, we need to warn it for users.

Reply:
Yes, that will be great